src/features/projects/runbooks/runbookRetentionPeriod.ts

```diff
@@ -1,4 +1,5 @@
 export interface RunbookRetentionPeriod {
     QuantityToKeep: number;
     ShouldKeepForever: boolean;
+    Unit: string;
```

Comment:
Can we introduce another type here to represent the available values for this field?